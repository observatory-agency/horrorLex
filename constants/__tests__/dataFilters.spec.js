const dataFilters = require('../dataFilters');

describe('dataFilters', () => {
  it('should match the snapshot', () => {
    expect(dataFilters).toMatchSnapshot();
  });
});
