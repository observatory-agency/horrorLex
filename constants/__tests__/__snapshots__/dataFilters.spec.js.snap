// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dataFilters should match the snapshot 1`] = `
Object {
  "propToArray": Object {
    "Films Discussed": "
",
    "ISBN-13": "
",
    "Tags": ", ",
  },
}
`;
